src/styles/card.scss

```diff
@@ -0,0 +1,61 @@
+@import 'variable';
+@import 'mixin';
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  @include card;
+
+  &__title {
+    @include card-title;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    .card__title {
+      color: $card-hover-title-color;
+    }
+  }
+
+  &__logo {
+    @include card-logo;
+  }
+
+  &__subtitle {
+    @include card-subtitle;
+  }
+
+  &__rating {
+    @include card-rating;
+  }
+
+  &__text-rating {
+    @include card-text-rating;
```

Comment:
<img width="489" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/70220667/87777302-1dd1-4fa4-bd4a-ffd6f94c510b">
it looks very strange, in a mixin, we usually bring out some styles that can be reused in several selectors, and not just transfer all the styles of the selector to it